server/game/cards/04.2-NTB/Consecration.js

```diff
@@ -0,0 +1,39 @@
+const SpellCard = require('../../spellcard.js');
+
+class Consecration extends SpellCard {
+    setupCardAbilities() {
+        this.spellAction({
+            title: 'Consecration',
+            playType: 'cheatin resolution',
+            target: {
+                activePromptTitle: 'Select your dude to Consecrate',
+                waitingPromptTitle: 'Waiting for opponent to select a dude',
+                cardCondition: { location: 'play area', controller: 'current'},
+                cardType: ['dude']
+            },
+            difficulty: 7,
+            actionContext: { card: this, gameAction: 'increaseBullets' || 'increaseInfluence' || 'setAsStud' },
```

Comment:
The `gameAction` does not work that way (with "or" ||). I saw the discussion on discord, so you can remove the `actionContext` here completely. If some effect is prevented, it will be so no need for `actionContext`